app/javascript/selectors/portfolioSelector.js

```diff
@@ -0,0 +1,28 @@
+export function getPortfolioName(portfolioId, state) {
```

Comment:
Can this file be renamed to `portfoliosSelector`? Just to follow the convention from our other selector files (the Redux reducer is called portfolios => portfoliosSelector)